homeassistant/components/modbus/base_platform.py

```diff
@@ -311,7 +311,10 @@ async def async_added_to_hass(self) -> None:
         """Handle entity which will be added."""
         await self.async_base_added_to_hass()
         if state := await self.async_get_last_state():
-            self._attr_is_on = state.state == STATE_ON
+            if state.state == STATE_ON:
+                self._attr_is_on = True
+            else:
+                self._attr_is_on = False
```

Comment:
Is there a change missing here? It seems like unknown will still be False?

Reply:
Correct somehow my added test removed a commit, corrected.